resources/compass/values.yaml

```diff
@@ -0,0 +1,34 @@
+global:
+  images:
+    containerRegistry:
+      path: eu.gcr.io/kyma-project/incubator
+    connector:
+      dir: pr/
+      version: "PR-59"
+    director:
+      dir: pr/
+      version: "PR-79"
+    gateway:
+      dir: pr/
+      version: "PR-59"
+    healthchecker:
```

Comment:
If dummy container, we should remove that from chart hmm?

Reply:
Let's keep it - we would need to add it again soon 🙂